WeatherForecast/WeatherForecast/HourlyWeatherInfo.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  TableViewCell.swift
+//  WeatherForecast
+//
+//  Created by 오승기 on 2021/10/11.
+//
+
+import UIKit
+
+class HourlyWeatherInfo: UITableViewCell {
```

Comment:
보통 UI 컴포넌트들은 네이밍에 자신의 identity를 나타내 주면 좋습니다
ex, WeathreTableViewCell, WeatherCustomView
```suggestion
class HourlyWeatherInfoCell: UITableViewCell {
```